src/www/messages/en.json

```diff
@@ -4,6 +4,7 @@
   "auto-connect-dialog-detail": "Unless you disconnect from your Outline server, Outline will automatically connect next time you restart your device.",
   "auto-connect-dialog-title": "Stay protected, always",
   "cancel": "cancel",
+  "change-language": "Change Language",
```

Comment:
Please update `resources/master_messages.json` with the new strings.

Reply:
Done